docs/reference/sql/functions/aggs.asciidoc

```diff
@@ -161,6 +284,9 @@ Returns the minimum value across input values in the field `field_name`.
 include-tagged::{sql-specs}/docs.csv-spec[aggMin]
 --------------------------------------------------
 
+[NOTE]
+`MIN` on a field of type <<text, `text`>> or <<keyword, `keyword`>> is translated into <<sql-functions-aggs-first>>.
```

Comment:
And I'm assuming MIN/MAX on keywords cannot be used in `HAVING`s...

Reply:
Yes, that's why I made the link here. Do you think I should add the info about HAVING here too?

Reply:
Yes, please.